### PR TITLE
Window popup

### PR DESCRIPTION
**Pull request by Gunjuzone**
In this pull request, javascript is used to create and append pop up window to my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -99,10 +104,11 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
 					<li class="btn-pro">Ruby</li>
 				</ul>
 	
-				<button class="btn-see-project">See Project</button>
+				<button class="btn-see-project modal-btn">See Project</button>
```
Comment (teekaytech) [inline index.html]: - [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears. This should be the same behavior for all `See Project` buttons in your portfolio. However, this is not true for the `See Project` button of the featured project: when I clicked on the button, the pop-up did not display. Kindly fix this :blush: 
- [ ] This is also an issue for other projects: on mobile screen sizes, the `See Project` popup is only working for the first project after the featured project, other buttons are not. Kindly fix this such that the popup will appear for all projects when the button is clicked.

pi

- [ ] You need to ensure that the popup follows the design for both desktop and mobile versions from the chosen template in 
  - Figma. Looking at the popup from the Figma design, there's a dark transparent background that comes up with the pop-up. 
  - Also, when the popup appears, scrolling the page should not scroll past the popup.
  - Please look at the Figma design below, compare it with yours and make necessary adjustments.

**_Figma Design_**
Screenshot from 2021-06-30 13-56-27

**_Your Design_**
clone

- [ ] 

**OPTIONAL**
- [ ] Your page is having a horizontal bar on all screen sizes. I believe this is caused by one of your containers. I strongly suggest you identify and fix this issue.

File: index.js [hunk 2]
```diff
@@ -9,4 +9,45 @@ function closeMenu() {
...
+data.appendChild(dataValue);
+
+const popClose = document.querySelector('.pop-up-close');
+popClose.addEventListener('click', () => {
+  dataValue.classList.add('displayModal');
+});
+
+modalBtn.addEventListener('click', () => {
+  dataValue.classList.remove('displayModal');
+});
```
Comment (teekaytech) [inline index.js]: - [ ] In order to have the popup for each of the projects with their dynamic contents, you need to implement your JavaScript code in such a way that there will be an array of objects to store JavaScript objects containing the information for each project.

- [ ] Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. i.e:
```JS
const projects = [
  {
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
  }
  ...
]
```

- [ ] You can then create the projects section dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in your existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

- [ ] Modal popup data is then populated with the data from the projects array. Following this process will make it easy to add projects in the future. It means that you will only add it in one place - as another object in the projects array.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Gunjuzone :wave:

_Your effort in this project is highly commendable :clap:_

- Linter checks (HTML+CSS+JS) is running with no errors :100: 
- PR title and summary looks professional :+1: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**